crypto/evp/p_rsa_asn1.c

```diff
@@ -192,3 +218,30 @@ const EVP_PKEY_ASN1_METHOD rsa_asn1_meth = {
 
   int_rsa_free,
 };
+
+const EVP_PKEY_ASN1_METHOD rsa_pss_asn1_meth = {
+  EVP_PKEY_RSA_PSS,
+  // 1.2.840.113549.1.1.10
+  {0x2a, 0x86, 0x48, 0x86, 0xf7, 0x0d, 0x01, 0x01, 0x0a}, 9,
+
+  rsa_pss_pub_decode,
+  NULL /* pub_encode */,
+  rsa_pub_cmp,
+
+  rsa_pss_priv_decode,
+  NULL /* priv_encode */,
+
+  NULL /* set_priv_raw */,
+  NULL /* set_pub_raw */,
+  NULL /* get_priv_raw */,
+  NULL /* get_pub_raw */,
+
+  rsa_opaque,
+
+  int_rsa_size,
+  rsa_bits,
+
+  0,0,0,
+
+  int_rsa_free,
```

Comment:
`RSA_free` might need to be modified if we are parsing more data for PSS. Haven't looked in detail.

Reply:
Yes, new PR of adding pss parameters to rsa struct will include pss parameters free.